louis/db/api/__init__.py

```diff
@@ -44,3 +70,48 @@ def match_documents_from_text_query(cursor, query):
     docs = match_documents(cursor, data['embedding'])
 
     return docs
+
+
+def search(cursor, query_embedding):
+    """Match documents with a given query."""
+    data = {
+        'text': ' '.join(sys.argv[1:]),
+        'query_embedding': query_embedding,
+        'match_threshold': 0.5,
+        'match_count': 1,
+        'weights': json.dumps(parsed_json)
+    }
+
+    cursor.execute("""
+        SELECT * 
+        FROM search(%(text)s, %(query_embedding)s::vector, %(match_threshold)s,
+                   %(match_count)s, %(weights)s::JSONB)
+    """, data)
+    # turn into list of dict now to preserve dictionaries
+    return [dict(r) for r in cursor.fetchall()]
+
+# Encode the query before doing the search
```

Comment:
use a docstring instead: https://peps.python.org/pep-0257/#one-line-docstrings

Reply:
@melanie-fressard 

1) you marked this as resolved instead of letting me resolve me - this is a no, no
2) you haven't actually fixed this, the comment is still not a function docstring